apps/wallet/src/routes/Deposit/Buy.tsx

```diff
@@ -6,18 +6,33 @@ import { MainButton } from '@twa-dev/sdk/react';
 import { useNavigate, useParams } from 'react-router-dom';
 import { Page } from '../../components';
 import { URLParams } from '../../types';
+import { colors } from '@telegram-wallet-ui/twa-ui-kit';
+import { getDebug } from '../../lib/utils/debug';
+
+const debug = getDebug('Buy');
 
 const walletAddress = account?.address;
 
+const convertToTransakCurrencyCode = (tokenSymbol: string) => {
+  switch (tokenSymbol) {
+    case 'wmatic':
+      return 'MATIC';
+    default:
+      return tokenSymbol.toUpperCase();
+  }
+};
+
 const constructSrcUrl = (walletAddress: string, tokenSymbol: string) => {
+  const cryptoCurrencyCode = convertToTransakCurrencyCode(tokenSymbol);
+
   const params = new URLSearchParams({
     network: 'polygon',
-    fiatCurrency: 'USD',
-    defaultFiatAmount: '30',
```

Comment:
`defaultFiatAmount` does not work with `fiatCurrency` unfortunately. They do now warn you in the Transak UI when you try to enter too low an amount 
<img width="436" alt="image" src="https://github.com/orbs-network/telegram-wallet-ui/assets/22128129/cf408e94-694a-41b7-bffd-7630d3138dd0">